modules/renter/renter.go

```diff
@@ -68,7 +68,9 @@ func New(cs *consensus.State, hdb modules.HostDB, wallet modules.Wallet, saveDir
 	// need to make sure that scanAllFiles() didn't get called until the entire
 	// balance had loaded, which would require loading the entire blockchain.
 	// This also won't be a problem once we're also saving the addresses.
-	r.scanAllFiles()
+	//
+	// TODO: bring back this functionality when we have resumable uploads.
+	//r.scanAllFiles()
```

Comment:
I don't think that this ever worked well in the first place, but it's definitely something we need. If you turned off the renter at 7 out of 12 completed uploads, then when you turn it back on it should try those 5 remaining ones again.

I guess this is sufficient for now though. I'm looking forward to the overhaul.